src/agent/communicator/tests/communicator_test.cpp

```diff
@@ -34,11 +34,20 @@ namespace
             .set_payload_claim("exp", jwt::basic_claim<jwt::traits::nlohmann_json>(exp))
             .sign(jwt::algorithm::hs256 {"secret"});
     }
+
+    const auto FUNC = []<typename T>([[maybe_unused]] const std::string&,
```

Comment:
Couldn't this just return T{}? instead of trying to return a time_t? is it needed for these tests? Also, the name be changed to GetConfigFunc or something like that?

Is it needed at all? It was nullptr before.

Reply:
Yes, we need to define this function as it can't be nullptr.